### PR TITLE
Adding codeonwers file

### LINKED ISSUE #51: Add `CODEOWNERS` file to allow doc team to be notified of when GCP bucket is updated
**Description**
For each release on this repository, a new public bucket is published on GCP, and the address to this bucket must be updated in the documentation.
To ensure that the documentation team is aware of this change, a `CODEOWNERS`file should be added.

**Disadvantage**
The granularity of the `CODEOWNERS` feature is at the file level, so, unfortunately, the @meilisearch/docs-team will be notified for every modification on the main file on this repo.

**To do**
- [ ] Give write access to @meilisearch/docs-team
- [ ] Create `CODEOWNERS` file at the root

### PR DESCRIPTION
**Pull request by alallema**
## Related issue
Fixes #51 

⚠️ This PR should be merged before the next release in order to avoid notifications due to the latest improvements to be finished in this repo.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Allow the documentation team to be informed when the PCB bucket address is updated.
+meilisearch.pkr.hcl @meilisearch/docs-team
```
Comment (bidoubiwa) [inline .github/CODEOWNERS:2]:
```suggestion
meilisearch.pkr.hcl @meilisearch/docs-team

```

I hope this does not break though

Comment (alallema) [reply .github/CODEOWNERS:2]: 😄 I don't know, I followed the GitHub documentation exactly and checked against [other repos.]
(https://github.com/meilisearch/documentation/blob/main/.github/CODEOWNERS)

Comment (bidoubiwa) [reply .github/CODEOWNERS:2]: Nop

**Review by bidoubiwa — APPROVED** [review]

**Comment by alallema** [comment]
bors merge

**Pull request merged**